app/models/alert.rb

```diff
@@ -9,6 +9,7 @@ class Alert < ApplicationRecord
 
   ALERT_LEVELS = [ 50, 80, 90, 100, 120, 150, 200, 300 ]
   VIOLATION_LEVEL = 100
+  TTL = 3.months
```

Comment:
what is TTL?

Reply:
"time to live" - how long the alerts will "live" in the DB